docs/beginner/tutorial3-pipeline/README.md

```diff
@@ -27,7 +27,7 @@ In order to do that, we're going to need something to do the conversion. Add the
 ```toml
 [dependencies]
 # ...
-glsl-to-spirv = "0.1"
+cshader = "0.6"
```

Comment:
Ah, I was trying to fix the same place in #45 (I closed it in favor of this PR). Maybe `shaderc`?

Reply:
Oops, right. Fixed in bd84d927665120cb439e773e98dc2c70f444092f.